turk-one/.idea/jsLibraryMappings.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="JavaScriptLibraryMappings">
+    <file url="PROJECT" libraries="{meteor-packages-auto-import-browser}" />
```

Comment:
You have to go through each of our collections that you want to import and inspect (workerTicket, solution, quitSurvey) and import them into the database one-by-one.

If you want to see any data live in the tool while doing your analysis, you should try this soon so you are comfortable swapping data in and out of your local deployment. 

In case you haven't done anything with Mongo yet from your local, here are the basics.

**to access the mongo shell**
+ start meteor
+ open a new terminal window and enter the following:
```
meteor mongo
```

Meteor will then enter the mongo shell for your running meteor instance in that terminal window. You can run queries exactly like we do in helpers to work with our data. For example, the following query will return our solutions printed in organized JSON.
```
db.solutions.find().pretty()
```

**to import from the turk-one exports**
+ start meteor
+ clear the collections you want to import for using steps above
+ navigate to the folder that the zip backups are in. Open a zip, and launch a terminal window with the inside of an unarchived zip as the context, so when you hit ```ls``` you see every [collectionName].json file.
+ open a new terminal window and enter the following
```
mongoimport --host localhost:3001 --db meteor --collection workerTickets --type json --file workerTickets.json
```

This one, for example, imports the workerTickets.json file to the workerTickets collection in the running database. Rinse and repeat for other collections.

**If you get an error that says 'mongoimport' is not recognized, you need to install mongoDB standalone from the mongo site. That should include the mongo tools like mongoimport, mongoexport, etc.**